plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcPageSourceFactory.java

```diff
@@ -257,7 +257,7 @@ private ConnectorPageSource createOrcPageSource(
         boolean originalFilesPresent = acidInfo.isPresent() && !acidInfo.get().getOriginalFiles().isEmpty();
         try {
             TrinoFileSystem fileSystem = fileSystemFactory.create(session.getIdentity());
-            TrinoInputFile inputFile = fileSystem.newInputFile(path);
+            TrinoInputFile inputFile = fileSystem.newInputFile(path, estimatedFileSize);
```

Comment:
From javadoc
```
     * The length will be returned from {@link TrinoInputFile#length()} and the actual file length
     * will never be checked. The file location path cannot be empty, and must not end with a slash
     * or whitespace.
```
can we use `estimatedFileSize` here?

Reply:
I think so, since the parquet reader is also doing this